ChaosMod/Memory/Hooks/DrawingHooks.cpp

```diff
@@ -0,0 +1,83 @@
+#include <stdafx.h>
+
+#include "Hook.h"
+#include "Util/Color.h"
+
+// Thanks CitizenFX!
+// Adapted by MoneyWasted for ChaosModV
+// https://github.com/citizenfx/fivem/blob/master/code/components/extra-natives-five/src/Draw2dNatives.cpp
+
+// This file is manually formatted.
+// clang-format off
+
+static const int MAX_LINES = 5000;
```

Comment:
Nitpick but could be a macro instead to be consistent with the way the project defines constants